src/widgets/survey/ui/Intermediate.tsx

```diff
@@ -60,13 +63,23 @@ function Intermediate({
     select: r => mapActivitiesFromDto(r.data.result.activities),
   });
 
-  const totalActivities = activityFlow!.activityIds.length;
```

Comment:
useAppletDetailsQuery above should be excluded and all dependent code should relay only on pipleline
add a comment - todo at least

Reply:
Added a comment